src/Entity/User.php

```diff
@@ -188,6 +206,15 @@ public function removeProgramme(Programme $programme): self
         return $this;
     }
 
+    public function addRole(string $role): self
+    {
+        if (in_array($role, $this->roles)) {
```

Comment:
it's a good practice to always add the third argument to in array as `true` to ensure strict types everywhere